hack/e2e/run-rp-and-e2e.sh

```diff
@@ -64,10 +59,6 @@ register_sub() {
       "https://localhost:8443/subscriptions/$AZURE_SUBSCRIPTION_ID?api-version=2.0"
 }
 
-run_e2e() {
```

Comment:
This file is only for dev, and CI=true is set when we call make test-e2e.  So I think we can remove
```
export E2E_CREATE_CLUSTER=true
export E2E_DELETE_CLUSTER=true
```
from this file.

Reply:
I have a very strong feeling if we remove this we gonna fail the same as with `AZURE_FP_CLIENT_ID` not being set. In old RPs we set this in PROD too and prod CI refactor expect these to create a cluster. We will be able to remove once rollout is done

Reply:
OK, let's leave it for later

Reply:
Add this to one of the KTLO items please